cookbooks/foundation/recipes/board.rb

```diff
@@ -43,3 +43,9 @@
   group node[:mediawiki][:group]
   mode 0o644
 end
+
+mediawiki_extension "MobileFrontend" do
+  site new_resource.site
```

Comment:
That will need to be the name of the site (ie `"board.osmfoundation.org"`) not `new_resource.site` which only makes sense in the context of the resource where it was before.

Reply:
Thanks I missed that. Updated.